back/lib/finder/sortable.rb

```diff
@@ -73,6 +73,12 @@ def _sort_with_method
                  end
     end
 
+    def _sort_with_attribute
+      attribute = _sort_method_suffix.include?('.') ? _sort_method_suffix : "#{table_name}.#{_sort_method_suffix}"
+
+      @records = order(attribute => _sort_order)
```

Comment:
`_sort_with_attribute` was not storing the ordered array in the `@records` instance variable